framework/src/org/apache/cordova/ConfigXmlParser.java

```diff
@@ -46,9 +46,14 @@ public CordovaPreferences getPreferences() {
     }
 
     public String getLaunchUrl() {
-        if (launchUrl == null) { 
+        if (launchUrl == null) {
```

Comment:
Nice work!
But, just a quick question.

There is also a setStartUrl method. The launchUrl is also prefixed with https there.
https://github.com/apache/cordova-android/pull/1222/files#diff-a32677abc0f97a96abd2948fad9d6b12f1251645a9f330431862dc83dabaeae2R150

Should it be prefixed with file if the AndroidInsecureFileModeEnabled returns true?

Reply:
You are right. I just pushed another change with this.